### DISCUSSION TITLE
CMI customrecipe compatibility problem and suggestion

### DISCUSSION THREAD

**Discussion by Fargus-Van-Crystal-de-Mindark** [Zrips/CMI#5031] [Enhancement]
Hello, hope to find you well, I have a compatibility problem in game... And I would like to describe to you the situation and suggest a solution that could help a lot of players... I hope that you understand what I will describe, because I'm not good in english.

Ok thanks for listening... :)

in CMI there is a feature /cmi customrecipe that let you create new recipes. But it is only minecraft edition... I would like to use IA items and slimefun (player heads) too to create new crafting recipes... Do you think that this will be possible?
Itemsadder has a free open api they can use without even owning the plugin at, all it's available here https://github.com/LoneDev6/API-ItemsAdder

https://itemsadder.plugin.ga/developers/java-api/examples
**This explains how to get itemsadder items easily with 1 line code**

Could you try to look at this? This will be wonderfool if you can let players use this feature without installing other plugins that do the exactly the same if we can already use CMI.

**Comment by Fargus-Van-Crystal-de-Mindark** [comment]
@Zrips it is possible to add this compatibility?

**Reply by CloudeLecaw** [reply]
No its not ;) it would take too much time, so its too much afford Just because may one user may need and will use it.;)

**Comment by Fargus-Van-Crystal-de-Mindark** [comment]
Please? Why I need to install another recipe controler if we can use the CMI one? please be open to custom model data... ♥

**Reply by CloudeLecaw** [reply]
It will take some years before he may has the time to refill your wish. Just dont give Up ;-*

**Reply by Fargus-Van-Crystal-de-Mindark** [reply]
this doesn't not make any sense, you know ? :) everyone is asking for adding custom model data on it

**Comment by Fargus-Van-Crystal-de-Mindark** [comment]
there are some news?

**Comment by CloudeLecaw** [comment]
Not really ;)
